crates/analyzer/src/handlers/create_symbol_table.rs

```diff
@@ -607,9 +616,36 @@ impl<'a> VerylGrammarTrait for CreateSymbolTable<'a> {
 
     fn enum_item(&mut self, arg: &EnumItem) -> Result<(), ParolError> {
         if let HandlerPoint::Before = self.point {
-            let value = arg.enum_item_opt.as_ref().map(|x| *x.expression.clone());
+            let value = if let Some(ref x) = arg.enum_item_opt {
+                let mut evaluator = Evaluator::new();
+                let evaluated = evaluator.expression(&x.expression);
+                if let Evaluated::Fixed { value, .. } = evaluated {
+                    EnumMemberValue::ExplicitValue(*x.expression.clone(), Some(value as usize))
+                } else {
+                    EnumMemberValue::ExplicitValue(*x.expression.clone(), None)
+                }
+            } else if let Some(ref x) = self.enum_member_value {
+                let previous_value = match x {
+                    EnumMemberValue::ExplicitValue(_expression, evaluated) => {
+                        if let Some(value) = evaluated {
+                            value
+                        } else {
+                            //  TODO:
+                            //  report error
+                            unreachable!();
```

Comment:
Generally `unreachable!` is used for "this statement is unreachable logically".
If reporting error is required in the future, `todo!("report error");` is better.